skimage/metrics/_variation_of_information.py

```diff
@@ -51,7 +51,7 @@ def _xlogx(x):
 
     Parameters
     ----------
-    x : ndarray or scipy.sparse.csc_matrix or csr_matrix
+    x : ndarray or scipy.sparse.csc_array or csr_array
```

Comment:
```suggestion
    x : ndarray or scipy.sparse.csc_array
```
The duplication seems like a typo? Not that it is that critical in an internal function.